src/components/gamespage/GamesPageProgressSection/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+import { Progress, Row, Col } from "antd";
+import { QuestionOutlined } from "@ant-design/icons";
+import recycleIcon from "../../../images/recycle.svg";
+
+const ProgressSection = ({
+  // XP stands for experience
+  // number
```

Comment:
Default parameters are for testing. I didn't create the logic yet. Logic will be inside container. Once I create container, I will delete default parameters.

Reply:
for testing like unit tests? just pass the parameters then. but also in general "I am committing this just for testing" = not allowed (because people forget to clean it up, all the time. no matter how perfect you think you are ;) )

Reply:
You are right. I'm deleting default parameters

Reply:
PTAL. I will fix others in next hours.